views/shop/index.hbs

```diff
@@ -19,10 +33,18 @@
             <div class="price pull-left">${{ this.price }}</div>
             <a href="/add-to-cart/{{this._id}}" class="btn btn-success pull-right" role="button">Add to Shopping Cart</a>
           </div>
+          <div class="card-body d-flex flex-column">
+            <h1 class="card-title pricing-card-title text-center">${{ this.price }}</h1>
+            <ul class="list-unstyled mt-3 mb-4">
+              <li>{{ this.description }}</li>
+            </ul>
+
+            <a href="/add-to-cart/{{this._id}}" class="btn mt-auto btn-success " role="button">Add to Shopping Cart</a>
+
+
         </div>
-      </div>
-    </div>
-  </div>
-    {{/each}}
+        </div>
+        {{/each}}
```

Comment:
Can be beatified?

Reply:
not needed

Reply:
is needed when you have time, this will be final version